javascript/javascript3/week3/githubPromises.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+function getUsers() {
+  let urls = [
+    "https://api.github.com/search/repositories?q=user:benna100",
+    "https://api.github.com/search/repositories?q=user:DeepthiDukka",
+    "https://api.github.com/search/repositories?q=user:gebremedhin85"
+  ];
+
+let requests = urls.map((url)=>fetch(url));
+Promise.all(requests).then(responce =>
+  responce.forEach((responce)=>console.log(responce.url))
+  //Extract the repositories
+  //render the repositories
+)};
+
+getUsers()
+
+const user1 = fetch(
+  "https://api.github.com/search/repositories?q=user:benna100"
+);
+const user2 = fetch(
+  "https://api.github.com/search/repositories?q=user:DeepthiDukka"
+);
+const user3 = fetch(
+  "https://api.github.com/search/repositories?q=user:gebremedhin85"
```

Comment:
This is not a good approach to save URL for each user as a variable. Imagine you would have 20 users. You could do
const url = 'https://api.github.com/search/repositories?q=user:`, and then add usernames.